### PR TITLE
Added css bundle from scss with webpack

### PR DESCRIPTION
**Pull request by danielmocan**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -69,8 +70,20 @@ module.exports = {
...
+                        "css-loader",
+                        "sass-loader",
+                    ],
+                } ),
+            },
         ],
     },
 
-    plugins,
+    plugins: [
```
Comment (alexnm) [inline webpack.config.js]: you are overwriting the plugins which are defined above

File: webpack.config.js [hunk 2]
```diff
@@ -69,8 +70,20 @@ module.exports = {
                 exclude: /node_modules/,
                 use: "babel-loader",
             },
+            {
+                test: /\.scss$/,
+                use: ExtractTextPlugin.extract( {
```
Comment (alexnm) [inline webpack.config.js:76]: do you need the extract text plugin here?

**Review by alexnm — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 2 above)
Comment (danielmocan) [reply webpack.config.js:76]: Yes, all the examples that I have seen have the options declared in module -> rules. Including the show cases the ExtractTextPlugin repository has on github.

File: webpack.config.js (hunk 1 above)
Comment (danielmocan) [reply webpack.config.js]: I moved ExtractTextPlugin to plugins variable

**Review by alexnm — APPROVED** [review]

**Pull request merged**